lvms-operator/lvmcluster.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: crc-lvmcluster
+  namespace: openshift-storage
+spec:
+  storage:
+    deviceClasses:
+    - name: rhel
+      fstype: xfs
+      default: true
+      deviceSelector:
+        paths:
+        - /dev/vda5
```

Comment:
Did you try using `/dev/disk/by-label/topolvm` to match what is done in `98-topolvm-partition.yaml`? This would make us less dependent on the actual disk layout.

Reply:
yes, tried and this works